canonical-web-and-design/ubuntu.com
# Swap webpack with esbuild

## Done

- Build JS with esbuild instead of webpack.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check JS across the site works well.
